Representable/representable
# lots of smallish frontend changes

**changes**
general
- [x] change css for links to be slightly different color
- [x] change large communities to be slightly less transparent

home page
- [x] fix background image on home page (slight error)
- [x] fix image of people on home page having slight inconsistencies
- [x] more padding above “OUR PARTNERS”
- [x] adds new button for “view and export”

footer
- [x] change logo in footer to be the word “rep” + logo maybe (if it looks good)
- [x] more padding on bottom of footer
- [x] change footer link — “partner profile” -> “organization profile”
- [x] add more links to pages in the footer

state selection
- [x] add margin to the left of the state name on dropdown box
- [x] change dropdown box to be blue outline, no background? — maybe like boxes on home page (olivia will design)
- [x] slightly less padding on top of “select your state”

education page
- [x] change images to the resized ones (from olivia)
- [x] change youtube to image that opens youtube link in a separate tab
- [x] more margins between image and dropdowns of state info

entry page
- [x] step 1: more left margin for text to align with dropdown “why do we need this information?”
- [x] step 2: change “What is a community of interest?” box to be a dropdown, rather than a full box
- [x] step 2: less padding above “name your community” section / box
- [x] tags modal (and maybe all the other modals) :: make X a little bigger and closer to the corner

mapping entry page
- [x] fix “edit info” button
- [x] look into changing “selection size” button to be more harmonious with other butotn designs
- [x] try to reduce whitspace between navbar and map

submission page
- [x] widen text directly under “congratulations” for thanks page
- [x] move buttons on thanks modal closer together
- [x] community info ?:
    - [x] add more padding on left
    - [x] move header text to align with dropdowns

drive landing page
- [x] get rid of next button
- [x] less padding between 1,2,3 and drive info
- [x] remove gradient from white -> light blue
- [x] keep get started button at landing, but add an arrow/chevron which scrolls to the drive info portion
- [x] make people image and “mapping america” text smaller
- [x] remove some padding from top of “get started” button at bottom

Survey Page
- [x] Spell out example on the survey page for each text box
- [x] Do bullet points instead of lines

Name you Community
- [x] Take out punctuation on the name your community part
- [x] Write in that you could type in an answer

**to test**
- visit each of these pages
- check that the changes are as expected, look for mistakes, etc.
